tests/test_integrity.py

```diff
@@ -127,6 +128,16 @@ def test_names_table(db):
 
     # Verify that each Source contains an entry in Names with Names.source = Names.other_source
     counts = db.query(db.Names.c.source).filter(db.Names.c.source == db.Names.c.other_name).distinct().count()
```

Comment:
Sorry to be pedantic, but can you use more descriptive variable names for the two `counts` variables? Line 126 and  130.